sdk/monitor/Microsoft.OpenTelemetry.Exporter.AzureMonitor/src/ApplicationInsightsRestClient.cs

```diff
@@ -46,16 +51,23 @@ internal async Task<int> InternalTrackAsync(ReadOnlyMemory<byte> body, Cancellat
             return message.TryGetProperty("ItemsAccepted", out var objItemsAccepted) && objItemsAccepted is int itemsAccepted ? itemsAccepted : 0;
         }
 
+        /// <summary>
+        /// Builds the ingestion Uri. (Example: https://dc.services.visualstudio.com/v2/track).
+        /// </summary>
+        internal RequestUriBuilder GetIngestionUri()
+        {
+            var uri = new RawRequestUriBuilder();
+            uri.AppendRaw(this.host, false);
+            uri.AppendPath($"/v{this.ApiVersion}/track", false);
```

Comment:
Depending on how often this API will get called, we might want to optimize out the string interpolation to reduce GC pressure.

Reply:
I agree. Currently, this gets called for every batch of telemetry items. 
I was on the fence about trying to optimize this vs making the change and moving on.
I can instead create this string on the apiVersion setter.